Makefile

```diff
@@ -3,17 +3,23 @@ SHELL := bash
 .DEFAULT_GOAL := build
 .DELETE_ON_ERROR:
 
+# Project directories
+testdata_dir := $(shell realpath ./test/testdata)
+script_dir := $(shell realpath ./test/scripts)
+template_dir :=  $(shell realpath ./test/templates)
+rendered_template_dir := $(shell mktemp -d -t bky-basm-rendered.XXXXXX)
+
 # add easyjson source file targets here
-easyjson_sources := basm/dto.go
-easyjson_generated := $(easyjson_sources:.go=_easyjson.go)
+easyjson_src := basm/dto.go
+easyjson_out := $(easyjson_src:.go=_easyjson.go)
```

Comment:
I was being somewhat inconsistent with my var naming and position in the makefile. I updated some var names to be consistent (uses src for source files that targets depend on, out for generated files) and moved project-level variables such as project directories to the top.

Reply:
I undid these changes as the gardening was starting get distracting